cdap-common/src/main/java/co/cask/cdap/common/logging/NamespaceLoggingContext.java

```diff
@@ -31,12 +31,19 @@ public NamespaceLoggingContext(final String namespaceId) {
   }
 
   @Override
+  public String getNamespacedPathFragment(String logBaseDir) {
```

Comment:
should rename this so it is clear it's the namespaced base dir. sounds like it's a namespaced version of getLogPathFragment()